ci/generate-and-upload-build-steps.sh

```diff
@@ -22,11 +23,30 @@ generate_build_configuration_steps () {
     # See https://docs.unrealengine.com/en-US/Programming/Development/BuildConfigurations/index.html for possible configurations 
     ENGINE_COMMIT_HASH="${1}"
 
+    if [[ -z "${NIGHTLY_BUILD+x}" ]]; then
+        export MACHINE_TYPE_ENVVAR="quad-high-cpu"
+    else
+        export MACHINE_TYPE_ENVVAR="single-high-cpu" # nightly builds run on smaller nodes
```

Comment:
I'm curious about why we introduce this difference. Are some configurations much more intensive to build?

Reply:
Ah, didn't read properly here